docs/deployment/restorePDMultiRegionSeedFailure.md

```diff
@@ -0,0 +1,164 @@
+
+Running the `status` command on one of the surviving pods, you will see in the output that there are configured servers in the topology that are not reachable:
+
+```
+          --- Mirrored Subtrees ---
+Base DN               : Write Master                   : Configured      : Outbound        : Inbound         : Failed Write
+                      :                                : Peers           : Connections     : Connections     : Operations
+----------------------:--------------------------------:-----------------:-----------------:-----------------:----------------
+cn=Cluster,cn=config  : N/A (single instance topology) : 0               : 0               : 0               : 0
+cn=Topology,cn=config : No Master (data read-only)     : 3               : 1               : 1               : 0
+```
+
+You will also see administrative alerts in the output indicating that the mirrored subtree manager can't establish a connection with the servers in the seed region.
+
+## Overview
+
+The key steps to restore the topology in this case are:
+
+1. Force a server in the surviving region to act as master of the topology
+2. Remove the unreachable servers from the topology
+3. Undo forcing a server to act as master of the topology
+4. Ensure any seed region pods are in single-server topologies
+5. Use dsreplication enable to add the servers from the refreshed seed region to the topology of the surviving region
+6. Use dsreplication initialize-all from a server of the surviving region to update the data across the regions
+
+## Force a server to act as master of the topology
+
+The PingDirectory topology will see that it cannot connect with half the servers and will switch to read-only mode. To allow the changes we need to make to the topology to fix this, exec into one of the pods in the surviving region.
+
+```
+kubectl exec -ti example-pingdirectory-0 sh
+```
+
+Run the following command to force this pod as master:
+
+```
+dsconfig set-global-configuration-prop --set force-as-master-for-mirrored-data:true --no-prompt
+```
+
+## Remove the unreachable servers from the topology
+
+Now we must tell the surviving pods that the original seed region pods no longer exist, and that they must be removed from the topology. These commands may take a long time to run, as the `remove-defunct-server` tool will keep trying to connect for up to ten minutes depending on the state of the seed region.
+
+```
+remove-defunct-server --ignoreOnline --serverInstanceName example-pingdirectory-1.west --bindDN [bind dn] --bindPassword [bind password]
+```
+
+In the above command, replace the --serverInstanceName argument with the instance name of one of the seed region pods. Then repeat the command for each seed region pod's instance name.
```

Comment:
replace the `--serverInstanceName` argument .. pods.  Repeat the command ... (no need for 'then')